content/desktop/hardened-desktop/enhanced-container-isolation/config.md

```diff
@@ -0,0 +1,249 @@
+
+```console
+$ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock docker:cli
+docker: Error response from daemon: enhanced container isolation: docker socket mount denied for container with image "docker.io/library/docker"; image is not in the allowed list; if you wish to allow it, configure the docker socket image list in the Docker Desktop settings.
+```
+This ensures malicious containers don't gain access to the Docker Engine, as
+such access could allow them to perform supply chain attacks (e.g., build and
+push malicious images into the organization's repositories) or similar.
+
+However, some legitimate use-cases require containers to have access to the
+Docker Engine socket. For example, the popular [TestContainers](https://testcontainers.com/)
+framework sometimes bind-mounts the Docker Engine socket into containers to
+manage them, perform post-test cleanup, etc.
+
+Starting with Docker Desktop 4.27, admins can optionally configure ECI to allow
+bind mounting the Docker Engine socket into containers, but in a restricted way.
+
+This can be done via the `admin-settings.json` file, as described in
+[Settings Management](../settings-management/configure.md). For example:
+
+```json
+{
+  "configurationFileVersion": 2,
+  "enhancedContainerIsolation": {
+    "locked": true,
+    "value": true,
+    "dockerSocketMountRestrictions": {
+      "imageList": {
+        "images": [
+          "docker.io/localstack/localstack:*",
+          "docker.io/testcontainers/ryuk:*"
+        ]
+      },
+      "commandList": {
+        "type": "deny",
+        "commands": ["push"]
+      }
+    }
+  }
+}
+```
+
+As shown above, there are two configurations for bind-mounting the Docker
+socket into containers: the `imageList` and the `commandList`. These are
+described below.
+
+#### Image List
+
+The `imageList` is a list of container images that are allowed to bind-mount the
+Docker socket. By default the list is empty (i.e., no containers are allowed to
+bind-mount the Docker socket when ECI is enabled). However, an admin can add
+images to the list, using either of these formats:
+
+| Image Reference Format  | Description |
+| :---------------------- | :---------- |
+| `<image_name>[:<tag>]`  | Name of the image, with optional tag; if tag is omitted, the `:latest` tag is used; if tag is the wildcard `*`, then it means "any tag for that image." |
+| `<image_name>@<digest>` | Name of the image, with a specific repository digest (e.g., as reported by `docker buildx imagetools inspect <image>`). This means only the image that matches that name and digest is allowed. |
+
+The image name follows the standard convention, so it can point to any registry
+and repository.
+
+In the example above, the image list was configured with two images:
+
+```json
+"imageList": {
+  "images": [
+    "docker.io/localstack/localstack:*",
+    "docker.io/testcontainers/ryuk:*"
+  ]
```

Comment:
looks like this snippet is missing a closing brace
```suggestion
  ]
}
```

Reply:
Thanks @dvdksn for catching that! Fixed.